Rubberduck.Core/UI/CodeExplorer/Commands/ExportCommand.cs

```diff
@@ -100,7 +100,15 @@ public bool PromptFileNameAndExport(QualifiedModuleName qualifiedModule)
                 var component = ProjectsProvider.Component(qualifiedModule);
                 try
                 {
-                    component.Export(dialog.FileName);
+                    if (component.Type == ComponentType.Document)
```

Comment:
Shouldn't we also do this for user forms? We have a special export for them due to the newline addition bug.